packages/storybook8/stories/Components/VideoTile/snippets/StylingVideoTile.snippet.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  CameraButton,
+  ControlBar,
+  FluentThemeProvider,
+  StreamMedia,
+  VideoTile,
+  VideoTileStylesProps,
+  DEFAULT_COMPONENT_ICONS
+} from '@azure/communication-react';
+import { initializeIcons, registerIcons, Stack } from '@fluentui/react';
+import React, { useState } from 'react';
+import { renderVideoStream } from '../../../utils';
+
+initializeIcons();
+registerIcons({ icons: DEFAULT_COMPONENT_ICONS });
```

Comment:
As we discussed moving this into the `manager.tsx`

Reply:
Tried moving to `manager.tsx`, it is throwing error, keeping it as it is for now, will keep investigating further.